app/experimenter/templates/experiments/section_analysis.html

```diff
@@ -7,4 +7,27 @@
 {% block section_content %}
   <p><strong>Data Science Owner: {{ experiment.analysis_owner }}</strong></p>
   {{ experiment.analysis|urlize|linebreaks }}
+
+  <p>
+  {{ experiment.SURVEY_REQUIRED_LABEL }}
```

Comment:
Formatting here is still a bit wonky:

<img width="250" alt="Screenshot 2019-05-22 11 25 36" src="https://user-images.githubusercontent.com/119884/58187203-5976dc80-7c84-11e9-91b6-86e21ab2b2c2.png">

"Is a survey required?" should be `<strong>` and I think you can get rid of the `<br>` altogether?

Reply:
ok got it